storage/mysql/log_storage.go

```diff
@@ -101,6 +102,7 @@ func createMetrics(mf monitoring.MetricFactory) {
 	queuedCounter = mf.NewCounter("mysql_queued_leaves", "Number of leaves queued", logIDLabel)
 	queuedDupCounter = mf.NewCounter("mysql_queued_dup_leaves", "Number of duplicate leaves queued", logIDLabel)
 	dequeuedCounter = mf.NewCounter("mysql_dequeued_leaves", "Number of leaves dequeued", logIDLabel)
+	clearedStmtCacheCounter = mf.NewCounter("mysql_cleared_prepared-statement_caches", "Number of times the prepared-statement cache has been cleared")
```

Comment:
Might be worth noting in the description that this is an unexpected condition / could indicate a problem.